IFC 4.3/IfcAlignment_IfcCourse_IfcSectionedSolidHorizontel/IfcAlignment_IfcCourse_IfcSectionedSolidHorizontel.ifc

```diff
@@ -0,0 +1,56 @@
+ISO-10303-21;
+HEADER;
+FILE_DESCRIPTION (('Export IFC'), $);
+FILE_NAME ('IFC_Export.ifc', '2021-01-15T07:21:22', (), ('AKG Software Consulting GmbH'), $, $, $);
+FILE_SCHEMA (('IFC4X3'));
+ENDSEC;
+DATA;
+#1 = IFCORGANIZATION('AKG Software Consulting GmbH', $, $, $, $);
+#2 = IFCAPPLICATION(#1, 'Build58', 'VESTRA INFRAVISION', $);
+#3 = IFCOWNERHISTORY($, #2, $, $, $, $, $, 1610691682);
+#4 = IFCDIMENSIONALEXPONENTS(0, 0, 0, 0, 0, 0, 0);
+#5 = IFCUNITASSIGNMENT((#6, #7, #8));
+#6 = IFCSIUNIT(*, .LENGTHUNIT., $, .METRE.);
+#7 = IFCSIUNIT(*, .TIMEUNIT., $, .SECOND.);
+#8 = IFCSIUNIT(*, .PLANEANGLEUNIT., $, .RADIAN.);
+#9 = IFCAXIS2PLACEMENT3D(#10, $, $);
+#10 = IFCCARTESIANPOINT((0., 0., 0.));
+#11 = IFCGEOMETRICREPRESENTATIONCONTEXT($, 'Model', 3, 1.E-6, #9, $);
+#12 = IFCPROJECTEDCRS('EPSG:25832', 'ETRS89_UTM32', 'EPSG:25832', 'EPSG:25832', 'UTM', '32', #6);
+#13 = IFCMAPCONVERSION(#11, #12, 0., 0., 0., $, $, $);

```

Comment:
not sure about this but if I've gotten it right: EPSG 25832 has an intended use of
> Europe between 6°E and 12°E: Austria; Belgium; Denmark - onshore and offshore; Germany - onshore and offshore; Norway including - onshore and offshore; Spain - offshore. 
[see here](https://epsg.io/25832)

Your Eastings and Northings in IfcMapConversion might not fit the above-mentioned reference.